### PR TITLE
merge 2020 updates into master

### PR DESCRIPTION
**Pull request by Daniel-E-B**

### CODE REVIEW

File: commands/Noop.java [hunk 1]
```diff
@@ -1,27 +1,15 @@
 package org.usfirst.frc4904.standard.commands;
 
 
-import edu.wpi.first.wpilibj.command.Command;
 
-public class Noop extends Command {
+public class Noop extends CustomCommand {
```
Comment (NikhilSuresh24) [inline commands/Noop.java]: commandbase

File: commands/Noop.java [hunk 2]
```diff
@@ -1,27 +1,15 @@
...
 	public Noop() {
 		super("Noop");
```
Comment (NikhilSuresh24) [inline commands/Noop.java]: `setName("Noop");`

File: custom/CommandSendableChooser.java [hunk 3]
```diff
@@ -16,8 +16,8 @@
 	 *
 	 * @param object
 	 */
-	public void addObject(Command object) {
```
Comment (NikhilSuresh24) [inline custom/CommandSendableChooser.java]: take in a `CommandBase`

**Comment by aemino** [comment]
I'm guessing you forgot to create a separate branch for this?

**Comment by Daniel-E-B** [comment]
> I'm guessing you forgot to create a separate branch for this?

I knew we were going to have to make changes but I didn't realize the scope when I made this PR. The 2020-update branch was intended to be used as the "separate branch"

File: .gitignore [hunk 4]
```diff
@@ -10,3 +10,5 @@
```
Comment (Daniel-E-B) [inline .gitignore]: I swear to god please for build

**Comment by bcho04** [comment]
Green squares 📗 🟢 💚 🍏 🥗

**Pull request merged**